### PR TITLE
Handling User Account Authentication

### PR DESCRIPTION
**Pull request by hanwenzhang123**
## What I did
- Testing Security Config (WIP) 
- Adding 404 Error Page
- Improving UI

### CODE REVIEW

File: code/SoloSavingsApp/pom.xml [hunk 1]
```diff
@@ -41,17 +41,25 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-data-jpa</artifactId>
 		</dependency>
-		<dependency>
-			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter-data-jpa</artifactId>
-		</dependency>
```
Comment (hanwenzhang123) [inline code/SoloSavingsApp/pom.xml:47]: Remove this dependency due to duplication (see dependency above)

File: .idea/workspace.xml
Comment (hanwenzhang123) [inline .idea/workspace.xml:1]: Need to remove this file.

To do:
- add a .gitignore file

**Comment by WAftring** [comment]
Reviewing.

**Review by to-whitney — COMMENTED** [review]
looks good, confirmed with team that SecurityConfig will be updated once other changes are made to limit access to endpoints.

**Review by WAftring — APPROVED** [review]
Approve

**Pull request merged**